### PR TITLE
Support for vualto rnv plugin + cleaner drm overrides

### PR DESCRIPTION
**Pull request by hosek**
Allows easier rnv file overriding of ReactExoplayerView

### CODE REVIEW

**Review by matejpolak — APPROVED** [review]

**Review by sanderlooijenga — APPROVED** [review]

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 1]
```diff
@@ -1,529 +1,38 @@
...
-                    playerNeedsSource = false;
-
-                    eventEmitter.loadStart();
-                    loadVideoStarted = true;
-                }
+    protected DrmSessionManager<FrameworkMediaCrypto> buildDrmSessionManager(
+                UUID uuid,
+                String licenseUrl,
+                String[] keyRequestPropertiesArray,
+                String vualtoToken,
```
Comment (CHaNGeTe) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:28]: where is vualtoToken used?

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewBase.java [hunk 2]
```diff
@@ -0,0 +1,1534 @@
...
+import com.google.android.exoplayer2.upstream.DefaultHttpDataSourceFactory;
+import com.google.android.exoplayer2.upstream.HttpDataSource;
+import com.google.android.exoplayer2.util.Util;
+import com.google.android.exoplayer2.ui.PlayerControlView;
+import com.twentyfouri.media.offline.ExoPlayerDownloadService;
+import com.google.android.exoplayer2.ext.mediasession.MediaSessionConnector;
+import com.vualto.vudrm.widevine.AssetConfiguration;
+import com.vualto.vudrm.widevine.WidevineCallback;
+import com.vualto.vudrm.widevine.vudrm;
+import com.vualto.vudrm.HttpKidSource;
```
Comment (CHaNGeTe) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewBase.java:97]: Seems prepared but unused, or am I wrong?

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java (hunk 1 above)
Comment (hosek) [reply android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:28]: only in override, I was thinking about renaming it to just token - its not vualto specific and others drm can use it too

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewBase.java (hunk 2 above)
Comment (hosek) [reply android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewBase.java:97]: must be merge error as there should be no vualto specific imports!

**Pull request merged**